user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,464 @@
+appear in the top right corner of the page.
+
+|
+
+.. image:: ./img/roles/import.png 
+
+|
+
+This is necessary for users who need to import large sets of data into the system. For example, these may include sales 
+representatives or employees responsible for lead development.
+
+.. _admin-capabilities-jobs:  
+
+Jobs Management
+---------------
+
+If this capability is set to *"System"*, the users can see jobs that have been started in the system, as well as view 
+their current status and their performance log from the *"Job Queue*" and *"Sheduled tasks"* pages. Links to these pages are 
+available in the *"System"* menu.
+
+The *"Job Queue*" and *"Sheduled tasks"* pages are usually usef by system administrators.
+
+
+.. _admin-capabilities-mailchimp:  
+
+MailChimp Manipulations
+-----------------------
+
+This capability is only meaningful if the system has been integrated with MailChimp. 
+
+Such integration lets users utilize MailChimp as an email campaign engine as described in the guide on 
+:ref:`Integration with MailChimp <user-guide-mc-integration>`. So, the users can synchronize lists of contacts in OroCRM
+and MailChimp, use them to create email campaigns in MailChimp, import them to OroCRM and use the MailChimp campaign 
+statistics to analyze the efficiency of the campaign. 
+
+All of these operations will only be available within OroCRM if the capability has been set to *"System"*.
+
+The MailChimp intergation opportunities are usually  used by marketing associates and other managers responsible for 
+email campaigns.
+
+
+.. _admin-capabilities-config-entities:  
+
+Manage Configurable Entities
+----------------------------
+
+Many entities in OroCRM can be configured from the UI, as described in the
+:ref:`Entities guide <user-guide-entity-management-from-UI>`. The user can change the attachments settings, 
+define whether the entity should be displayed on a grid and/or a View page, whether it will be 
+exported to a .CSV file, and define other settings. For some of them, it is also possible to add new fields, as 
+described in the :ref:`Entity Fields guide <user-guide-field-management>`. 
+
+These actions are only available if the *"Manage Configurable Entities"* capability is set to *"System"*. 
+
+They are usually performed by the system administrators.
+
+
+.. _admin-capabilities-org-calendar-events: 
+
+Manage Organization Calendar Events
+-----------------------------------
+
+If this capability has been set to *"System"*, the user create, edit, and delete events in organization-wide calendars, 
+which are described in more detail in the :ref:`corresponding section <user-guide-calendars-system>` of 
+the *Calendars Overview* guide.
+
+Organization calendar events are usually managed by organization-level managers and HRs.
+
+.. hint::
+
+     Even if this capability is set to *"None"*, the users can still view organization-wide calendars, add 
```

Comment:
Edit: Even if this capability is set to *"None,"* the users can still view organization-wide calendars, add